perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/UsersManagerMethod.java

```diff
@@ -496,6 +496,35 @@ public UserExtSource call(ApiCaller ac, Deserializer parms) throws PerunExceptio
 		}
 	},
 
+	/*#
+	 * Gets list of all user's external sources with attributes. If attrsNames is empty
```

Comment:
If this version of the method doesn't specify ``attrsNames`` parameter, you shouldn't mention it in the general text.